hera_cal/xtalk_filter.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 the HERA Project
+# Licensed under the MIT License
+
+"""Module for delay filtering data and related operations."""
+
+import numpy as np
+
+from . import io
+from . import version
+from .vis_clean import VisClean
+
+import pickle
+import random
+import glob
+import os
+import warnings
+
+
+class XTalkFilter(VisClean):
```

Comment:
I think this should be a subclass of `delay_filter` and you shoudl leave the `get_filled_data` and `write_filled` data in the DelayFilter class

Reply:
`DelayFilter` by it's very name does delay filtering so I much prefer a separate `xtalk_filter` module. `get_filled` and `write_filled` can just be moved to `VisClean` where any other filtering modules inherit them (they are useful in many contexts).

Reply:
ah yes, I agree--I think this is probably fine